packages/web-components/@groupby/elements-autocomplete/test/unit/autocomplete.test.ts

```diff
@@ -455,4 +487,21 @@ describe('Autcomplete Component', () => {
       expect(dispatchElementsEvent).to.be.calledWith(UPDATE_SEARCH_TERM, payload);
     });
   });
+
+  describe('requestUpdateSearchTerm()', () => {
+    it('should dispatch an update search term event', () => {
+      stub(autocomplete, 'selectedItem').get(() => ({ label: 'dress' }));
+      const group = autocomplete.group = 'group-1';
+      const payload = {
+        term: autocomplete.selectedItem.label,
+        group,
+        search: false,
+      };
+      const dispatchElementsEvent = stub(autocomplete, 'dispatchElementsEvent');
+
+      autocomplete.requestUpdateSearchTerm(group);
```

Comment:
This does not take in `group` (anymore?). 
```suggestion
      autocomplete.requestUpdateSearchTerm();
```